modules/raft/src/main/java/org/apache/ignite/internal/raft/storage/LogStorageFactory.java

```diff
@@ -17,13 +17,17 @@
 
 package org.apache.ignite.internal.raft.storage;
 
+import static org.apache.ignite.internal.util.CompletableFutures.nullCompletedFuture;
+
+import java.util.concurrent.CompletableFuture;
 import org.apache.ignite.internal.close.ManuallyCloseable;
 import org.apache.ignite.internal.components.LogSyncer;
+import org.apache.ignite.internal.manager.IgniteComponent;
 import org.apache.ignite.raft.jraft.option.RaftOptions;
 import org.apache.ignite.raft.jraft.storage.LogStorage;
 
 /** Log storage factory interface. */
-public interface LogStorageFactory extends ManuallyCloseable, LogSyncer {
+public interface LogStorageFactory extends ManuallyCloseable, LogSyncer, IgniteComponent {
```

Comment:
Since this is now an `IgniteComponent`, we can remove the `ManuallyCloseable` inheritance